lib/stack-functions

```diff
@@ -270,8 +270,7 @@ stack-parameters() {
   aws cloudformation describe-stacks        \
     --stack-name ${stack}                   \
     --query "${query}"                      \
-    --output ${output:-json}                |
-    jq --sort-keys .
+    --output ${output:-json}
```

Comment:
IMO, this is a hack since it removes the functionality for the sake of supporting an edge case.  Perhaps, we can check for the $output variable and if it's empty or defined to be 'json' sort the output by keys?